packages/snowpack/src/config.ts

```diff
@@ -1,3 +1,5 @@
+import buildScriptPlugin from '@snowpack/plugin-build-script';
+import runScriptPlugin from '@snowpack/plugin-run-script';
```

Comment:
Question: do we still want to import these? I think we may need to for backwards compatibility, but wanted to confirm.

Reply:
Definitely, lets keep this backwards compat until at least v3